### PR TITLE
Fix links and format issues  for Analytics Doc

### PR DESCRIPTION
**Pull request by cacheung**
Fix links and format issues for Analytics Doc

### CODE REVIEW

**Comment by emdobrin** [comment]
Changes look good. Can we also get these links fixed in all occurrences ?
- Before you use this API, see [Identify unique visitors](https://experienceleague.adobe.com/docs/analytics/components/metrics/unique-visitors.html)
- Sets a custom Analytics visitor identifier. For more information, see [Custom Visitor ID](https://experienceleague.adobe.com/docs/analytics/implementation/vars/config-vars/visitorid.html).

**Comment by cacheung** [comment]
@emdobrin Good Catch.  Fixed the links.

File: using-mobile-extensions/adobe-analytics/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 1. In Experience Platform Launch, click the **Extensions** tab.
 2. On the **Catalog** tab, locate the **Adobe Analytics** extension, and click **Install**.
-3. Type the extension settings. For more information, see [Configure the Analytics Extension](./#configure-analytics-extension).
+3. Type the extension settings. For more information, see [Configure the Analytics Extension](README.md#configure-analytics-extension).
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics/README.md:7]: this link seems not to work - should be README.md#configure-the-analytics-extension

**Review by emdobrin — APPROVED** [review]
Tiny comment, looks good otherwise 👍

File: using-mobile-extensions/adobe-analytics/README.md (hunk 1 above)
Comment (cacheung) [reply using-mobile-extensions/adobe-analytics/README.md]: Thanks @emdobrin, I didn't notice it was missing "the".  Fixed it.  All links  should work now.  Can you help to merge?

**Pull request merged**